packages/serverpod_serialization/lib/src/serialization.dart

```diff
@@ -190,7 +190,11 @@ abstract class SerializationManager {
   /// Encode the provided [object] to a Json-formatted [String].
   /// If [formatted] is true, the output will be formatted with two spaces
   /// indentation.
-  static String encode(Object? object, {bool formatted = false}) {
+  static String encode(
+    Object? object, {
+    bool formatted = false,
+    bool checkIfIsProtocol = false,
```

Comment:
Suggestion: rename to `encodeForProtocol`.

I think that name better reflects what we are trying to do.